src/Charts/ChartListParams.js

```diff
@@ -7,6 +7,7 @@ class ChartListParams {
   constructor (filter = null, tag = null, expandEvents = null) {
```

Comment:
Wouldn't it be nice to have validation in the constructor with default  set to false for ease of use, and to keep it consistent with the other params?

Reply:
@namfal i've updated the constructor, and fixed the test.. can you take another look please?

Reply:
Looks good! (looked good before too, I am just being picky :) )